README.md

```diff
@@ -120,9 +120,9 @@ The server exposes a GraphQL API for all actions. The server is built with the f
 Amplication Client is the front end of the platform that provides you with an easy-to-drive UI for building your next low-code application.
 The client is based on React, Apollo client, Primer components, React Material Web Components, Formik, and more.
 
-# Version 1.0.0
+# Version 1.0.1
```

Comment:
Let's do this as Version 1 only so we don't have to change it on every minor and patch release.

Reply:
Made the change!